package.json

```diff
@@ -10,12 +10,15 @@
     "lint": "eslint index.js",
     "test:watch": "ava --watch",
     "test": "npm run lint && ava",
-    "prepublish": "npm run build && npm test"
+    "test:types": "tsd",
```

Comment:
How could we call both ava and tsd with the yarn test and yarn test:watch scripts?

Reply:
Not sure if I get what you mean. Are you wondering how to call both ava and tsd in watch mode at the same time with one call? However, `tsd` seems to miss a "watch" option